serverless/webpack.config.js

```diff
@@ -1,7 +1,36 @@
 const serverlessWebpack = require('serverless-webpack');
+const babelOptions = require('../babel.config');
+const path = require('path');
 
 module.exports = {
   entry: serverlessWebpack.lib.entries,
   target: 'node',
   mode: 'production',
+  module: {
+    rules: [
+      {
+        test: /\.(js|jsx|ts|tsx)$/,
+        exclude: /node_modules/,
+        use: {
+          loader: require.resolve('babel-loader'),
+          options: {
```

Comment:
check if necessary, I think it shouldn't be since default babel config resolution should work